### PR TITLE
103 info and help pages

### PR DESCRIPTION
**Pull request by AndyTWF**
- Add Info item that tells the user which version of UKCP they're using.
- Add help link that directs their web browser to the (upcoming) UKCP help pages on the api.

### CODE REVIEW

File: src/plugin/PluginHelpPage.h [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+                bool ProcessCommand(std::string command) override;
+
+                // The command to accept
+                const std::string command = ".ukcp help";
+
+                // The description of the menu item
+                const std::string menuItemDescription = "UKCP Help (Opens In Web Browser)";
+
+                // URL for the help
+                const std::wstring helpUrl = L"https://ukcp.vatsim.uk/web/help";
```
Comment (AxonC) [inline src/plugin/PluginHelpPage.h:37]: Is there a redirect to core in place from this already?

File: src/plugin/UKPlugin.cpp [hunk 2]
```diff
@@ -260,7 +260,7 @@ namespace UKControllerPlugin {
     {
         EuroScopePlugIn::CFlightPlan plan = this->FlightPlanSelect(callsign.c_str());
 
-        if (!plan.IsValid() || plan.GetSimulated()) {
+        if (!plan.IsValid()) {
```
Comment (AxonC) [inline src/plugin/UKPlugin.cpp:263]: Is this stuff meant to be in this change set?

**Review by AxonC — DISMISSED** [review]

File: src/plugin/UKPlugin.cpp (hunk 2 above)
Comment (AndyTWF) [reply src/plugin/UKPlugin.cpp:263]: It was in a previous release... but apparently has come up on this PR!

File: resource/resource.h [hunk 3]
```diff
@@ -1,124 +1,124 @@
...
-// 
-#ifdef APSTUDIO_INVOKED
-#ifndef APSTUDIO_READONLY_SYMBOLS
-#define _APS_NEXT_RESOURCE_VALUE        121
-#define _APS_NEXT_COMMAND_VALUE         40001
-#define _APS_NEXT_CONTROL_VALUE         1094
-#define _APS_NEXT_SYMED_VALUE           101
-#endif
-#endif
+﻿//{{NO_DEPENDENCIES}}
```
Comment (AxonC) [inline resource/resource.h:1]: I don't see what this changeset is? 👁️

**Review by AxonC — CHANGES_REQUESTED** [review]
There's a couple of suspect changesets you might want to look at

File: resource/resource.h (hunk 3 above)
Comment (AndyTWF) [reply resource/resource.h:1]: As the branch is old I think this is mostly Git complaining about newlines

**Comment by AndyTWF** [comment]
This is in main now

**Comment by AndyTWF** [comment]
.... no it isn't

**Review by AxonC — APPROVED** [review]

**Pull request merged**